shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -461,19 +474,43 @@ - (void)dispatchMouseEvent:(NSEvent*)event phase:(FlutterPointerPhase)phase {
   }
 }
 
-- (void)dispatchKeyEvent:(NSEvent*)event ofType:(NSString*)type {
+- (void)propagateKeyEvent:(NSEvent*)event {
+  for (NSResponder* responder in self.additionalKeyResponders) {
+    if (event.type == NSEventTypeKeyDown) {
+      if ([responder respondsToSelector:@selector(keyDown:)]) {
+        [responder keyDown:event];
+      }
+    } else {
+      if ([responder respondsToSelector:@selector(keyUp:)]) {
+        [responder keyUp:event];
+      }
+    }
+  }
+}
+
+- (void)dispatchKeyEvent:(NSEvent*)event {
+  if (event.type != NSEventTypeKeyDown && event.type != NSEventTypeKeyUp) {
+    return;
```

Comment:
It looks like you've broken the `flagsChanged:` method below.

Reply:
Ahh, I didn't see that it could be called with events other than those two.

OK, I put back the `ofType` parameter, and handle `NSEventFlagsChanged` as well now.